vcpkg/consume/manifest-mode.md

```diff
@@ -0,0 +1,321 @@
+---
+title: "Tutorial: Install a dependency from a manifest file"
+description: Learn to install your projects dependencies with vcpkg by using a manifest file.
+author: vicroms
+ms.author: viromer
+ms.date: 8/10/2023
+ms.prod: vcpkg
+ms.topic: tutorial
+---
+# Tutorial: Install a dependency from a manifest file
+
+vcpkg has two operation modes: classic mode, and manifest mode. This article describes how to install packages using manifest mode, which is the recommended workflow for most users.
+
+In manifest mode, you declare your project's direct dependencies in a manifest file named
+[`vcpkg.json`](../reference/vcpkg-json.md).
+
+Manifest files have their own `vcpkg_installed` directory where they install their dependencies,
+unlike classic mode, where all packages are installed in a common `%VCPKG_ROOT%/installed`
+directory. Manifests are suitable for using vcpkg with multiple codebases, as each codebase can have
+a different set of dependencies that do not interfere with each other.
+
+Manifest mode is also required to use advanced features like
+[versioning](../users/versioning.md) and [custom registries](../users/registries.md).
+
+In this tutorial, you learn how to:
+
+> [!div class="checklist"]
+> * [Create a project with a manifest]
+> * [Install dependencies]
+> * [Build the project]
+
+## Create a project with a manifest
+
+For this tutorial, we use the following source file:
+
+`main.cxx`:
+
+:::code language="cxx" source="../examples/snippets/manifest-mode-cmake/main.cxx":::
+
+The code references the open-source libraries: `cxxopts`, `fmt`, and `range-v3`; which are all
+available in the vcpkg public registry at <https://github.com/Microsoft/vcpkg>.
+
+To satisfy these dependencies, create a file named `vcpkg.json` in the project's folder:
+
+`vcpkg.json`:
+
+:::code language="json" source="../examples/snippets/manifest-mode-cmake/vcpkg.json":::
+
+You only need to specify your direct dependencies in the `"dependencies"` list. When it runs, vcpkg
+resolves and installs any required transitive dependencies.
+
+## Install the dependencies
+
+### [MSBuild](#tab/msbuild)
+
+> [!div class="nextstepaction"]
+> [Learn more about using vcpkg from MSBuild](../users/buildsystems/msbuild-integration.md)
+
+### Enable user-wide integration (MSBuild)
```

Comment:
I think some of these h3's should be promoted to h2's with steps and any relevant sub-steps should be enumerated and nested within the h2 section.